doc/modules/mod_pubsub.md

```diff
@@ -0,0 +1,67 @@
+### What is PubSub?
+
+PubSub is a fascinating design pattern which mostly promotes a loose coupling between two kinds of entity (publishers and subscribers).
+Like their names suggest, in the pubsub world we have publishers who fire events, and subscribers who wish to be notified about those events when publishers push data.
+There might be several subscribers, several publishers, and even several channels (or nodes) where the events are sent.
+
+### Module Description
+
+This module implements [XEP-0060 (Publish-Subscribe)](http://www.xmpp.org/extensions/xep-0060.html).
+Due to the complexity of the protocol, the PubSub engine makes successive calls to `nodetree` and `node plugins` in order to check the validity of requests, perform the corresponding action and return a result or appropriate error.
+Such an architecture makes it much easier to write custom pubsub plugins and add new storage backends.
+It's all about fitting PubSub to your needs!
+
+### Options
+
+* `host` (string, default: `"pubsub.@HOST@"`): Subdomain for Pubsub service to reside under.
+`@HOST@` is replaced with each served domain.
+* `access_create` (atom, default: `all`): Who is allowed to create pubsub nodes.
+* `max_items_node` (integer, default: `10`): Define the maximum number of items that can be stored in a node.
+* `max_subscriptions_node` (integer, default: `undefined` - no limitation): The maximum number of subscriptions managed by a node.
+* `nodetree` (binary, default: `<<"tree">>`): Specify the storage and organisation of the pubsub nodes.
```

Comment:
Do we really expect binaries in the pubsub's configuration? Usually the config file contains ordinary strings.